pkgs/servers/asterisk/default.nix

```diff
@@ -78,14 +79,9 @@ let
     };
   };
 
-  pjproject_2_7_1 = fetchurl {
```

Comment:
why not just fetch this from github https://github.com/pjsip/pjproject/releases?

Reply:
I wanted it to be exactly the file it expects. There was a hash somewhere iirc